mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from odoo import fields, models
+
+
+class MgmtsystemAction(models.Model):
+    _inherit = "mgmtsystem.action"
+
+    def _complete_name(self):
```

Comment:
```suggestion
    def _compute_complete_name(self):
```

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models

> Method conventions
Compute Field: the compute method pattern is _compute_<field_name>
Inverse method: the inverse method pattern is _inverse_<field_name>
Search method: the search method pattern is _search_<field_name>
Default method: the default method pattern is _default_<field_name>
Onchange method: the onchange method pattern is _onchange_<field_name>
Constraint method: the constraint method pattern is _check_<constraint_name>